.github/workflows/release.yml

```
@@ -0,0 +1,13 @@
+name: Build executable binaries and release them
+
+on:
+  push:
+    tags:
+      - 'v*'
+      - 'test-'
+  
```

yamllint

[error] trailing spaces

(trailing-spaces)